src/generate.ts

```diff
@@ -42,7 +42,8 @@ export const generate = (datadir: string) => {
           decimals: token.overrides?.decimals ?? data.decimals,
           logoURI: `${BASE_URL}/data/${folder}/logo.${logoext}`,
           extensions: {
-            optimismBridgeAddress: token.overrides?.bridge ?? NETWORK_DATA[chain].bridge,
+            optimismBridgeAddress:
+              token.overrides?.bridge ?? NETWORK_DATA[chain].bridge,
```

Comment:
```suggestion
              token.noBridge ? undefined : token.overrides?.bridge ?? NETWORK_DATA[chain].bridge,
```
Is this necessary?

Reply:
I think we should leave this as-is, since below this `out.extensions.optimismBridgeAddress` is deleted when noBridge is true